deploy/dashboards/metrics/README.md

```diff
@@ -2,7 +2,20 @@
 
 The following dashboards are for use with Open Liberty using mpMetrics-1.x or mpMetrics-2.x.
 
-Choose the dashboards from the Grafana dashboard directory that matches your installation.
+## Dashboard Installation 
+### OKD and Grafana
+- Download the JSON dashboard file that matches your mpMetrics version from the `OKD3.11-Grafana5.2` folder.
+- Open Grafana.
+- Click the Create [+] icon > Click "Import" > Click "Upload .json file" > Upload.
 
+### RHOCP and Grafana Operator with OLM
 
+- The YAML dashboard file can be installed using the `oc apply` command in the project containing Grafana Operator.
+- Switch to the project containing Grafana Operator i.e. `oc project grafana`.
+- Navigate this repository to the YAML dashboard file that you wish to install and copy the URL generated by the "Raw" button as an argument to the `-f` flag. For example, for RHOCP and Grafana Operator 3.0.2 running Open Liberty mpMetrics-2.x, you can run:
+    ```
+    oc apply -f https://raw.githubusercontent.com/OpenLiberty/open-liberty-operator/master/deploy/dashboards/metrics/RHOCP4.3-GrafanaOperator3.0.2-Grafana5.2/open-liberty-grafana-mpMetrics2.x.yml
+    ```
 
+### RHOCP and Grafana Operator without OLM or Vanilla Grafana
+- Users deploying Grafana Operator without OLM *or* a vanilla install of Grafana, should refer to the installation steps and JSON dashboards provided for **OKD and Grafana**.
```

Comment:
this last bullet doesn't sound right -- user isn't in process of deploying Grafana Operator, they are deploying dashboards 😄 .  Also, I don't quite get the vanilla install comment.

Reply:
good point modified the first, also I guess the vanilla install doesn't align with our OCP metrics story, by that I mean, using a helm/manual install of Grafana, not the operator. So I have removed that part.